### DISCUSSION TITLE
Is it possible to see the differences between 3 files in all directions

### DISCUSSION THREAD

**Discussion by amid0** [WinMerge/winmerge#1709] [General]
As I understand correctly now in case if I'm opening 3 binary files I can see highlighted diff between 1-2 and 2-3 but cant see diff 1-3.
Is it possible to configure?
Or requires additional development?

**Comment by sdottaka** [comment]
Unfortunately additional development is required.

**Reply by amid0** [reply]
Ok.
Got it.
Unfortunately I'm .net dev and not having experience in C++ to try to add (
